### PR TITLE
Other regular patterns modified from labman to labcontrol

### PR DESCRIPTION
**Pull request by charles-cowart**
Includes copyright comment header
'LabmanSomething' class names
some misc instances of labman and/or Labman

### CODE REVIEW

File: labcontrol/db/configuration_manager.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 class ConfigurationManager(object):
-    """Holds the labman configuration
+    """Holds the labcontrol configuration
```
Comment (AmandaBirmingham) [inline labcontrol/db/configuration_manager.py]: Would this be "LabControl", or is there a reason for no?

File: labcontrol/db/testing.py [hunk 2]
```diff
@@ -36,10 +36,10 @@ def reset_test_db():
                      'AmmCWZuabe0O5Mp28s1')
     qclient = QiitaClient(
         "https://localhost:8383", client_id, client_secret,
-        server_cert=labcontrol.db.settings.labman_settings.qiita_server_cert)
+        server_cert=labcontrol.db.settings.labcontrol_settings.qiita_server_cert)
     qclient.post("/apitest/reset/")
     # The above call resets the qiita schema. Qiita does not create the
-    # labman structures, so create them here
+    # labcontrol structures, so create them here
     path_builder = partial(join, dirname(__file__), 'support_files')
```
Comment (AmandaBirmingham) [inline labcontrol/db/testing.py]: Same q as above

File: labcontrol/db/user.py [hunk 3]
```diff
@@ -99,12 +99,12 @@ def login(cls, email, password):
...
+        LabControlUnknownIdError
             Email is not recognized
-        LabmanLoginError
+        LabControlLoginError
             Provided password doesn't match stored password
-        LabmanLoginDisabledError
-            If the user doesn't have access to login into labman
+        LabControlLoginDisabledError
+            If the user doesn't have access to login into labcontrol
         """
```
Comment (AmandaBirmingham) [inline labcontrol/db/user.py]: Same q as above

File: labcontrol/db/configuration_manager.py (hunk 1 above)
Comment (charles-cowart) [reply labcontrol/db/configuration_manager.py]: Sure!

File: labcontrol/db/user.py (hunk 3 above)
Comment (charles-cowart) [reply labcontrol/db/user.py]: no problem

File: labcontrol/db/testing.py (hunk 2 above)
Comment (charles-cowart) [reply labcontrol/db/testing.py]: sure!

**Pull request merged**